src/lib/fetchData.ts

```diff
@@ -1,6 +1,7 @@
 import { NOCODB_BASE } from "$env/static/private";
 import { nocodb_api } from "$lib/nocodb_api";
-import { getRandomSubset } from "./utils";
+import type { Person } from "../models/person";
+import { toKebabCase, getRandomSubset, capitaliseFirstLetter } from "./utils";
```

Comment:
code seems to break on this model import. am I missing some build command to generate the types?

Reply:
added the missing import, thanks for the catch